CHANGELOG.md

```diff
@@ -3,14 +3,14 @@
 New features:
 
 * Context pre-initialization with TruffleRuby `--native`, which significantly
-  improves startup time and loads the did_you_mean gem ahead of time.
+  improves startup time and loads the `did_you_mean` gem ahead of time.
 
 Performance:
 
-* Dir.glob is much faster and more memory efficient in cases that can reduce
+* `Dir.glob` is much faster and more memory efficient in cases that can reduce
   to direct filename lookups.
 
-* SecureRandom now defers loading OpenSSL until it's needed, reducing time to
+* `SecureRandom` now defers loading OpenSSL until it's needed, reducing time to
   load SecureRandom.
```

Comment:
Do you want SecureRandom to be stylized here?

Reply:
Fixed.